Immense.RemoteControl.Desktop.Shared/Startup/CommandProvider.cs

```diff
@@ -57,7 +57,7 @@ public static class CommandProvider
         var sessionIdOption = new Option<string>(
            new[] { "-s", "--session-id" },
            "In Unattended mode, this unique session ID will be assigned to this connection and " +
-           "shared with the server.  The connection can then be found in the DesktopHubSessionCache " +
+           $"shared with the server.  The connection can then be found in the RemoteControlSessionCache " +
```

Comment:
Unnecessarily made this string interpolatable. You don't have to change this unless you're a stickler for stuff like this ;)

Reply:
Ah.  Nice catch.  I was going to use `nameof` but realized the class couldn't be referenced here.  I changed it.  :)